src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -32,24 +32,32 @@ using namespace ov;
 using namespace ov::frontend::tensorflow;
 
 namespace {
-std::vector<std::string> get_unconverted_types_from_model(const std::shared_ptr<Model>& model) {
-    std::vector<std::string> unconverted_ops_types;
+
+void get_unsupported_operations_and_failures(const std::shared_ptr<Model>& model,
+                                             std::vector<std::string>& unsupported_operations,
+                                             std::unordered_map<std::string, std::string>& failures) {
     for (const auto& node : model->get_ordered_ops()) {
         if (const auto& fw_node = ov::as_type_ptr<FrameworkNode>(node)) {
             auto op_type = fw_node->get_decoder()->get_op_type();
-            unconverted_ops_types.push_back(op_type);
+            auto fw_node_attrs = fw_node->get_attrs();
+            if (fw_node_attrs.find(FrameworkNode::failed_conversion_key) != fw_node_attrs.end() &&
+                failures.count(op_type) == 0) {
+                // save only the first encountered failure that is more improtant for developer
+                // that means the translator is found but the conversion is failed
+                failures[op_type] = fw_node_attrs.at(FrameworkNode::failed_conversion_key);
+            } else if (std::find(unsupported_operations.begin(), unsupported_operations.end(), op_type) ==
```

Comment:
could you use `unordered_set`? O(n) is not so good

Reply:
Now some tests rely on the order of unsupported operations. If I start use `set`, the order is lost for them. Proper structuring will be fixed 107500 ticket where I am going to store it in the `unordered_set` and save it in the exception for further communication with MO.